.buildkite/pipeline.yml

```diff
@@ -46,3 +46,28 @@ steps:
                   env:
                     - BUILDKITE_PULL_REQUEST=${BUILDKITE_PULL_REQUEST}
                     - GITHUB_PR_LABELS=${GITHUB_PR_LABELS}
+
+  - label: "Trigger x-pack/Auditbeat"
+    plugins:
+      - monorepo-diff#v1.0.1:
+          diff: "git diff --name-only HEAD~1"
+          watch:
+            - path:
+                - x-pack/auditbeat/
+                - .buildkite/x-pack/pipeline.xpack.auditbeat.yml
+                #XPACK
+                - go.mod
+                - pytest.ini
+                - dev-tools/
+                - libbeat/
+                - testing/
+                - x-pack/libbeat/"
+              config:
+                trigger: "beats-xpack-auditbeat"
+                label: ":package: xpack/auditbeat pipeline"
```

Comment:
```suggestion
                label: "Trigger xpack/auditbeat pipeline"
```

Reply:
<img width="238" alt="Screenshot 2024-04-10 at 23 03 20" src="https://github.com/elastic/beats/assets/3244018/5d518224-3397-4959-b31d-338928983f12">
It helps differentiate trigger steps from launched pipelines. Are you sure?

Reply:
I don't have a preference. I'm following what the other pipelines already have and want to keep them consistent. Let's not refactor it now.